draft-packages/illustration/KaizenDraft/Illustration/Scene.tsx

```diff
@@ -223,285 +229,449 @@ export const EmptyStatesNeutral = ({
 }
 
 export const IntroductionsNewAccount = (props: SceneProps) => (
-  <Base {...props} name="illustrations/scene/introductions-new-account.svg" />
+  <Base
+    aspectRatio={props.enableAspectRatio ? "portrait" : undefined}
```

Comment:
There isn't many portraits, but I've gone through and added this to the ones that are.